web/src/pages/index.js

```diff
@@ -1,142 +1,142 @@
-import React from 'react';
-import clsx from 'clsx';
-import Link from '@docusaurus/Link';
-import Layout from '@theme/Layout';
-import HomepageFeatures from '@site/src/components/HomepageFeatures';
-import styles from './index.module.css'
+import React from "react";
+import clsx from "clsx";
+import Link from "@docusaurus/Link";
+import Layout from "@theme/Layout";
+import HomepageFeatures from "@site/src/components/HomepageFeatures";
+import styles from "./index.module.css";
 
+import landingCyclops from "/static/img/landing_cyclops.png";
+import title from "/static/img/cyclops-title.png";
 
-import landingCyclops from '/static/img/landing_cyclops.png';
-import title from '/static/img/cyclops-title.png';
-
-import yaml from '/static/img/yaml_background.png';
+import yaml from "/static/img/yaml_background.png";
 
 import Comparison from "../components/Comparison";
 import Install from "../components/Install";
 import Features from "../components/Features";
 import Blogs from "../components/Blogs";
 import Testimonials from "../components/Testimonials";
-import {Button, Col, ConfigProvider, Row} from "antd";
+import Newsletter from "../components/Newsletter";
```

Comment:
There are only 2 actual changes (the rest is Prettier formatting).
The first change is here, importing the `Newsletter` component.